src/icon/index.tsx

```diff
@@ -56,6 +56,8 @@ import ShareIcon from './shareIcon'
 import SmokeIcon from './smokeIcon'
 import SpeakIcon from './speakIcon'
 import StarIcon from './starIcon'
+import TabBusIcon from './tabBusIcon'
```

Comment:
I don't think we should prefix them with `Tab`. The library is open source, so we shouldn't expect that these will always be used inside a tab

Reply:
Agreed but I'm reluctant to name it Car and Bus, knowing they are not what we actually use for cars and buses... If you have any other suggestions

Reply:
Alright I did remove `Tab`